syntaxes/vscode-scl.tmLanguage.json

```diff
@@ -43,11 +43,17 @@
 				{
 					"name": "keyword.control.scl",
 					"match": "(?i)\\b(and|or|xor|not)\\b"
+				},
+				{
+					"name": "constant.regexp.scl",
+					"match": "\\#.\\w*(?!\\()\\b[.]\\w.*?\\b|\\#\\D.\\w*(?!\\()\\b|\\.\\w*|\\.\\w*"
```

Comment:
what does this do? :)

Reply:
This one will correctly highlight any # variables used in the code https://regex101.com/r/O7UL43/1

Reply:
I see. Nice. 
But they are not constants. It should probably be some other keyword so that it makes sense for people wanting to change the highlighting in the settings. I don't remember entirely how this works. Maybe `"name": "variable.regexp.scl"` would work. And then we need to add to the top pattern of the syntax file?
`
"patterns": [
		{ "include": "#comments"},
		{ "include": "#section" },
		{ "include": "#keywords" },
		{ "include": "#strings" },
		{ "include": "#operators" },
		{ "include": "#storage"	},
		{ "include": "#constants" },
		{ "include": "#support"	},
		{ "include": "#punctuations" },
		{ "include": "#variables" }

	],
`

Reply:
Updated.